src/tools/genpolicy/src/config_map.rs

```diff
@@ -76,6 +76,27 @@ pub fn get_value(value_from: &pod::EnvVarSource, config_maps: &Vec<ConfigMap>) -
     None
 }
 
+//eg ["key1=value1", "key2=value2"]
+pub fn get_values(config_map_name: &str, config_maps: &Vec<ConfigMap>) -> Option<Vec<String>> {
+    for config_map in config_maps {
+        if config_map_name == &config_map.metadata.name.clone().unwrap() {
+            return config_map
```

Comment:
This expression is too complex for my brain - but I'm OK with it if you test it and it works.

Reply:
Refactored this to try to make it [a bit more clear](https://github.com/microsoft/kata-containers/pull/128/files#diff-316862490937e09fc44d29ee85ceca9a8dfeda9b1b299a4e8c4c05ba12b3b470R68-R79). Verified config values are made available inside the container as expected
<img width="526" alt="image" src="https://github.com/microsoft/kata-containers/assets/30801614/7f1c82dc-a638-4bf2-a748-d83a66c4d4a3">